### ISSUE TITLE
{Freeway-Bug}[Adaptive Card iOS - Activity Update]: Voice Over is conveying irrelevant information for the 'Publish Adaptive Card Schema' heading.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#24]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to ' ActivateUpdate.Jsons' and double tap to activate it.
7. The Card will open.
8. Perform right swipe, navigate to 'Publish Adaptive Card Schema' and observe the issue. 

**Actual Result:**
While navigating to the 'Publish Adaptive Card Schema' heading, screen reader is conveying irrelevant information. It is announcing as **'Publish Adaptive Card Schema heading text field group double tap to edit'** while it is non interactive field.

**Expected Result:**
Voice Over should convey the correct and relevant information for the 'Publish Adaptive Card Schema' and it should announce as **'Publish Adaptive Card Schema heading'**

**Note:**
This issue is repro throughout the application for the labels and headings.

**User Experience:**
It will impact the screen reader user as they will get confused if the screen reader does convey the incorrect information as group or double tap to activate it as they will try to perform action on it.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/24d64dfd-b0df-4b4a-9617-8a7b16167ee6

**Comment by vagpt** [comment]
Hi @JoshuaChingMSFT , @ardlank,

Could you please provide us with an updated build to test this issue? We have checked and recently we have the same version on the app center on which we have performed the Test Pass i.e. 1.0 (2.4.0-beta.23.10.15.1).

CC: @Raisul123

**Comment by vagpt** [comment]
This issue is still repro at our end hence reactivating this issue. Please find the attachment for the reference.

https://github.com/user-attachments/assets/12b45bd1-def3-457a-813f-bd2d10380584

**Comment by GabrielMedAlv** [comment]
No reproduced, validated with latest main branch. To be verified on next release.

https://github.com/user-attachments/assets/a3878c31-d094-4593-8198-6c9ec74a5a79
